applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -128,10 +129,9 @@ config BT_PACS_SNK_CONTEXT
 	hex
 	default 0x0005
 
-#TODO: This should be fixed in Zephyr since BT_AUDIO_CONTEXT_TYPE_PROHIBITED
-#      should be involved automatically when there's no audio source endpoint
 config BT_PACS_SRC_CONTEXT
 	hex
+	default 0x0005 if STREAM_BIDIRECTIONAL
 	default 0x0000
```

Comment:
The default in Zephyr for this is 0x03ff

Reply:
I think "default 0x0000" can be removed from here.
If I remember correctly, we added "default 0x0000" to avoid a build warning. But it seems this build warning is fixed now

Reply:
Done